Makefile

```diff
@@ -12,7 +12,7 @@ ORG := jenkins-x-plugins
 ORG_REPO := $(ORG)/$(NAME)
 RELEASE_ORG_REPO := $(ORG_REPO)
 ROOT_PACKAGE := github.com/$(ORG_REPO)
-GO_VERSION := 1.17.9
+GO_VERSION := 1.18.4
```

Comment:
Replace GO_VERSION with this section: https://github.com/jenkins-x-plugins/jx-pipeline/blob/89593c6e1e9d2c01b8a26296f53b6fdedcc48410/Makefile#L16-L22